src/clarity-angular/button-group/index.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2016 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Type } from "@angular/core";
+import {Button} from "./button";
+import {ButtonGroup} from "./button-group";
+
+export const BUTTON_GROUP_DIRECTIVES: Type<any>[] = [
+    ButtonGroup,
+    Button
+];
```

Comment:
General code style/convention question:

Do we want to have newline terminated files? I see some with and some without.